cgs_vmc/wavefunctions.py

```diff
@@ -1080,6 +1080,85 @@ def from_hparams(
     return cls(**full_vector_params)
 
 
+class GNN(Wavefunction):
```

Comment:
I think this could be more descriptive, considering that this is not a well-known architecture. e.g. `GraphConvNetwork`  (this also avoids abbreviations in class names, which is a plus)